src/preload.py

```diff
@@ -324,6 +325,14 @@ def resize_rootfs(additional_space):
     sfdisk(tmp.name, _in=new_layout)
     offsets_and_sizes = get_offsets_and_sizes(image, "s")
     copy = partial(ddd, _if=image, of=tmp.name, bs=SECTOR_SIZE, conv="notrunc")
+    # Copy data between the MBR and the first partition (some devices rely on
+    # the bootloader being there, like the Variscite DART-6UL)
+    mbr_size_in_sectors = MBR_SIZE // SECTOR_SIZE
```

Comment:
I could be wrong, but don't you want to `ceil()` this number, rather than `floor()`? If it takes 0.5 of another sector, surely we need to copy that sector as well? Or am I missing something?

Reply:
This relies on the fact that MBR_SIZE == SECTOR_SIZE == 512.
I will update this in the upcoming GPT support PR where the `dd`'s `bs` will be increased and the `seek`, `skip` and `count` will use bytes.

Reply:
Please could you add a comment detailing this then, that could definitely catch somebody out.